nrfcloud/settings.ts

```diff
@@ -21,14 +20,16 @@ export type Settings = {
 export const getSettings = ({
 	ssm,
 	stackName,
+	scope,
 }: {
 	ssm: SSMClient
 	stackName: string
+	scope: string
```

Comment:
```suggestion
	account: string
```

So in the rest of the codebase you can get rid of this:

```typescript
const scope = `thirdParty/${account}`
```